deis-dev/manifests/deis-workflow-manager-rc.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: deis-workflow-manager
+  namespace: deis
+  labels:
+    heritage: deis
+spec:
+  replicas: 1
+  selector:
+    app: deis-workflow-manager
+  template:
+    metadata:
+      labels:
+        app: deis-workflow-manager
+    spec:
+      serviceAccount: deis-workflow-manager
+      containers:
+      - name: deis-workflow-manager
+        image: quay.io/deisci/workflow-manager:v2-beta
+        imagePullPolicy: Always
+        env:
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: WORKFLOW_MANAGER_PORT
+          value: "8080"
+        - name: WORKFLOW_MANAGER_VERSIONSAPIURL
+          value: "https://versions.deis.com"
+        - name: WORKFLOW_MANAGER_STAGINGAPIURL
+          value: "https://versions-staging.deis.com"
+        - name: WORKFLOW_MANAGER_DOCTORAPIURL
```

Comment:
Yeah, it was to make incorporation with https://github.com/kelseyhightower/envconfig a bit easier.

E.g., here's its usage in `workflow-manager`:

`envconfig.Process("workflow_manager", &Spec)`

`Spec` is a pointer to this:

```
// Specification config struct
type Specification struct {
	Port           string `default:"8080"`
	Polling        int    `default:"43200"` // 43200 seconds = 12 hours
	VersionsAPIURL string
	DoctorAPIURL   string
	APIVersion     string
	CheckVersions  bool   `default:"true"`
	DeisNamespace  string `default:"deis"`
}
```

When I used underscores in the property names, my golang IDE was complaining that that isn't the go way.

That's the purpose!